datamodel/app/view/vw_tww_additional_ws.py

```diff
@@ -0,0 +1,576 @@
+          WHERE NE.fk_wastewater_structure = NEW.obj_id
+        );
+      END IF;
+
+      RETURN NEW;
+    END;
+    $BODY$
+    LANGUAGE plpgsql;
+
+
+
+    DROP TRIGGER IF EXISTS vw_tww_additional_ws_UPDATE ON tww_app.vw_tww_additional_ws;
+
+    CREATE TRIGGER vw_tww_additional_ws_UPDATE INSTEAD OF UPDATE ON tww_app.vw_tww_additional_ws
+      FOR EACH ROW EXECUTE PROCEDURE tww_app.ft_vw_tww_additional_ws_UPDATE();
+    """.format(
+        update_co=update_command(
+            pg_cur=cursor,
+            table_schema="tww_od",
+            table_name="cover",
+            table_alias="co",
+            prefix="co_",
+            indent=6,
+            skip_columns=["situation3d_geometry"],
+            remap_columns={"cover_shape": "co_shape"},
+        ),
+        update_sp=update_command(
+            pg_cur=cursor,
+            table_schema="tww_od",
+            table_name="structure_part",
+            table_alias="sp",
+            prefix="co_",
+            indent=6,
+            skip_columns=["fk_wastewater_structure"],
+            update_values={
+                "last_modification": "NEW.last_modification",
+                "fk_dataowner": "NEW.fk_dataowner",
+                "fk_provider": "NEW.fk_provider",
+            },
+        ),
+        update_ws=update_command(
+            pg_cur=cursor,
+            table_schema="tww_od",
+            table_name="wastewater_structure",
+            table_alias="ws",
+            remove_pkey=False,
+            indent=6,
+            skip_columns=[
+                "detail_geometry3d_geometry",
+                "last_modification",
+                "_usage_current",
+                "_function_hierarchic",
+                "_label",
+                "_cover_label",
+                "_bottom_label",
+                "_input_label",
+                "_output_label",
+                "fk_main_cover",
+                "fk_main_wastewater_node",
+                "_depth",
+            ],
+            update_values={},
+        ),
+        update_wt=update_command(
+            pg_cur=cursor,
+            table_schema="tww_od",
+            table_name="wwtp_structure",
+            table_alias="wt",
+            prefix="wt_",
+            remove_pkey=True,
+            indent=6,
+            skip_columns=[],
+            remap_columns={"obj_id": "obj_id"},
+        ),
+        update_sm=update_command(
+            pg_cur=cursor,
+            table_schema="tww_od",
+            table_name="small_treatment_plant",
+            table_alias="sm",
+            prefix="sm_",
+            remove_pkey=True,
+            indent=6,
+            skip_columns=[],
+            remap_columns={"obj_id": "obj_id"},
+        ),
+        update_dt=update_command(
+            pg_cur=cursor,
+            table_schema="tww_od",
+            table_name="drainless_toilet",
+            table_alias="dt",
+            prefix="dt_",
```

Comment:
this one should be `to` to be coherent with the OID?

Reply:
prefix only I suppose?

Reply:
yes, exactly